demos/projects/NXP/mimxrt1060/ADU.md

```diff
@@ -0,0 +1,256 @@
+
+```json
+"tags": {
+    "ADUGroup": "<your-tag-here>"
+},
+```
+
+Viewing the device twin on the portal, the "tag" section should look similar to the following. Don't worry if you do or do not have a `"deviceUpdate"` section in the `"ADUGroup"` tag. ADU adds that as a default group.
+
+![img](../../../../docs/resources/tagged-twin.png)
+
+### Prepare the Sample
+
+To connect the MIMXRT1060-EVK to Azure, you'll modify a configuration file for Azure IoT settings, rebuild the image, and flash the image to the device.
+
+Update the file `iot-middleware-freertos-samples/demos/projects/NXP/mimxrt1060/config/demo_config.h` with your configuration values.
+
+If you're using a device previously created in your **IoT Hub** with SAS authentication, disable DPS by commenting out `#define democonfigENABLE_DPS_SAMPLE` and setting the following parameters:
+
+Parameter | Value
+---------|----------
+ `democonfigDEVICE_ID` | _{Your Device ID value}_
+ `democonfigHOSTNAME` | _{Your Azure IoT Hub Host name value}_
+ `democonfigDEVICE_SYMMETRIC_KEY` | _{Your Primary Key value}_
+
+If you're using **DPS** with an individual enrollment with SAS authentication, set the following parameters:
+
+Parameter | Value
+---------|----------
+ `democonfigID_SCOPE` | _{Your DPS ID scope value}_
+ `democonfigREGISTRATION_ID` | _{Your Device Registration ID value}_
+ `democonfigDEVICE_SYMMETRIC_KEY` | _{Your Primary Key value}_
+
+### Build the Image
+
+To build the device image, navigate to the `iot-middleware-freertos-samples` directory and run the following commands:
+
+  ```bash
+    cmake -G Ninja -DBOARD=mimxrt1060 -DVENDOR=NXP -Bmimxrt1060 .
+    cmake --build mimxrt1060
+  ```
+
+After the build completes, confirm that a folder named `mimxrt1060/` was created and it contains a file named `demo/projects/NXP/mimxrt1060/iot-middleware-sample-adu.bin`.
+
+### Flash the Bootloader
+
+1. Connect the Micro USB cable to the Micro USB port on the NXP EVK, and then connect it to your computer. After the device powers up, a solid green LED shows the power status.
+
+1. Use the Ethernet cable to connect the NXP EVK to an Ethernet port.
+
+1. In File Explorer, find the bootloader at `demos/projects/NXP/mimxrt1060/bootloader/rt1060_sbl.bin` and copy it.
+
+1. In File Explorer, find the  NXP EVK device that's connected to your computer. The device appears as a drive on your system with the drive label `RT1060-EVK`.
+
+1. Paste the bootloader binary file into the root folder of the NXP EVK. Flashing starts automatically and completes in a few seconds.
+
+1. Reset the device once flashing completes.
+
+### Confirm Device Bootloader Details
+
+You can use one of the terminal applications to monitor communication and confirm that your device is set up correctly. Go to Device Manager in Windows to determine which COM port your device was assigned.
```

Comment:
I thought we have a command-line now updated in all samples to show how to detect COM ports. Could we use that one please?

Reply:
It looks like this only works for the Espressif devices :(